src/libraries/System.Text.Json/Common/JsonNumberHandling.cs

```diff
@@ -39,6 +39,8 @@ enum JsonNumberHandling
         /// will be written as their corresponding JSON string representations.
         /// Strings that have escaped characters will be unescaped before reading.
         /// Leading or trailing trivia within the string token, including whitespace, is not allowed.
+        ///
+        /// You can do this, but you need to be aware that this will create potentially non-interoperable JSON (see https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonnumberhandling).
```

Comment:
- This might be better as a remark.
- The link would be redundant since it would point from Intellisense to the exact same content on the docs website. A link would be more appropriate in an augmented exception message for the scenario highlighted in https://github.com/dotnet/runtime/issues/48926#issuecomment-788365962.
- Add the same remark to the `WriteAsString` entry above.

```suggestion
        /// <remark>
        /// You can do this, but you need to be aware that this will create potentially non-interoperable JSON.
        /// </remark>
```